content/docs/other-guides/usage-reporting.md

```diff
@@ -20,24 +20,28 @@ command only restarts spartakus with reportUsage set to `false`
 kubectl -n ${NAMESPACE} delete deploy -l app=spartakus
 ```
 
-**Reporting usage data is one of the most significant contributions you can make to Kubeflow; so please consider turning it on.** This data
-allows us to improve the project and helps the many companies working on Kubeflow justify continued investment.
+**Reporting usage data is one of the most significant contributions you can make to Kubeflow; so please consider turning it on.** This data allows us to improve the project and helps the many companies working on Kubeflow justify continued investment.
 
-You can improve the quality of the data by giving each Kubeflow deployment a unique id by editing `spartakus.yaml` in your `KF_APP` directory:
+**To prevent Spartakus from being deployed,** do the following before running `kfctl apply`:
+- Edit `${KFAPP}/app.yaml`
+- Delete the entry in KfDef.Spec.Applications for spartakus
 
 ```
-apiVersion: v1
-data:
-  usageId: "your usage id"
-kind: ConfigMap
-metadata:
-  labels:
-    kustomize.component: spartakus
-  name: spartakus-parameters
-  namespace: kubeflow
-```
-
-Then deploy your changes:
-```
-kubectl apply -f spartakus.yaml`
+  - kustomizeConfig:
+      parameters:
+      - initRequired: true
+        name: usageId
+        value: <randomly-generated-id>
+      - initRequired: true
+        name: reportUsage
+        value: "true"
+      repoRef:
+        name: manifests
+        path: common/spartakus
+    name: spartakus
 ```
+- Delete the entry in KfDef.Spec.ComponentParams for spartakus``` spartakus: 
+    * initRequired: true name: usageId value: 
+    * initRequired: true name: reportUsage value: ```
+- Delete the entry in KfDef.Spec.Components for spartakus```
```

Comment:
Remove this line - it looks like a duplicate.

Reply:
@sarahmaddox ,  In fact,  it is different sections in the yaml files, so I list it clearly here. Please review that.

Reply:
Following is the real format of the yaml file
  ```
spartakus:
    - initRequired: true
      name: usageId
      value: <randomly-generated-id>
    - initRequired: true
      name: reportUsage
      value: "true"
```

Reply:
Ah I see - it's clear now that the formatting is fixed. Thanks.